.github/workflows/main.yaml

```diff
@@ -48,15 +51,12 @@ jobs:
       - name: Set up node
         uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4.4.0
         with:
-          node-version: 20
+          node-version: ${{ env.NODE_VERSION }}
```

Comment:
[**question**] Shouldn't this be `${{ matrix.node-version }} `? And I believe we shouldn't have any env with a node version at the top otherwise it will override all the builds. See comment from rabbitai https://github.com/openfga/js-sdk/pull/247/files#r2269791797